src/main/java/org/hydev/mcpm/client/uninstallUseCase/Uninstaller.java

```diff
@@ -0,0 +1,36 @@
+package org.hydev.mcpm.client.uninstallUseCase;
+import java.io.File;
+import java.util.Arrays;
+import org.hydev.mcpm.client.injector.PluginLoader;
+import org.hydev.mcpm.client.injector.PluginNotFoundException;
+
+
+public class Uninstaller implements UninstallBoundary{
+    private boolean remove_depen;
+
+
+    public void uninstall(String name) throws PluginNotFoundException {
+        PluginLoader pil = new PluginLoader();
+        pil.unloadPlugin(name);
+
+        // Find plugin file
+        // could be useful for removing dependencies (?)
+        var plugins_folder = new File("plugins");
+        var plugins = Arrays.stream((plugins_folder.listFiles()))
+                .filter(plugin -> plugin.getName().endsWith(name+".jar"));
+
+        File file = new File(plugins.toArray()[0].toString());
```

Comment:
What if the plugin file is empty? Make sure edge cases and exceptions are handled whenever possible.

Reply:
I am thinking of making a class to handle the file system (maybe the local tracker can handle this) for installation, uninstall and update.

1. create a folder to store plugins
   - If a folder doesn't exist: 
      + Create one for installing (install)
      +  Send an error message for an update (since the plugin does not install yet) and uninstall (the plugin does not exist)
    - If it exists, do nothing

2.Check (and store) the file path of the (older version) plugins in the directory for uninstall and delete
    For updates, use uninstall and bypass the file path to install input => install. The parenthesis note is for the update.